packages/scrum-server/src/api/components/tips/controller.js

```diff
@@ -1,2 +1,17 @@
 // controller.ts
-// The controller class handles incoming requests, validates them and sends the response data back to the client. It uses the service class to interact with the database.
\ No newline at end of file
+// The controller class handles incoming requests, validates them and sends the response data back to the client. It uses the service class to interact with the database.
+const TipsService = require('./service');
+
+const tipFind = async (req, res) => {
+  try {
+    const serviceResponse = await TipsService.find(req);
+    return res.status(200).json({
+      success: true,
+      data: serviceResponse.data,
```

Comment:
We can do this to all apis 
`data`  instead of specific names ?  what do you think ? or why only tips?

Reply:
I think it's better across the services. I just did tips cause was the one I was focussing on 😅